sdk/tests/conformance2/transform_feedback/transform_feedback.html

```diff
@@ -315,6 +315,22 @@
     runVaryingsTest();
 }
 
+function verifyTransformFeedbackVarying(progrom, index, valid, name) {
```

Comment:
> progrom

I'm sure this works because program is a global variable - maybe rename it so the local doesn't shadow the global